packages/neovim/build.sh

```diff
@@ -82,7 +82,15 @@ termux_step_post_make_install() {
 	cp $TERMUX_PKG_BUILDER_DIR/sysinit.vim $_CONFIG_DIR/
 
 	# Tree-sitter grammars are packaged separately and installed into TERMUX_PREFIX/lib/tree_sitter.
-	ln -s "${TERMUX_PREFIX}"/lib/tree_sitter "${TERMUX_PREFIX}"/share/nvim/runtime/parser
+	ln -sf "${TERMUX_PREFIX}"/lib/tree_sitter "${TERMUX_PREFIX}"/share/nvim/runtime/parser
+
+	mv "${TERMUX_PREFIX}"/bin/nvim "${TERMUX_PREFIX}"/bin/nvim.bin
+	cat <<-EOF > "${TERMUX_PREFIX}"/bin/nvim
```

Comment:
I think this should be a separate file.
I'm also not the biggest fan of keeping the nvim binary in `/bin` if we're going to wrap it.

There's a couple other packages that get wrapped, I'd like to take a look at those first before making a final decision on if and how we should do the same for Neovim.

Reply:
Is there a good way to substitute `TERMUX_PREFIX` on install, though? That's why I chose the heredoc, but if there's a better way I'd gladly fix that.

libexec might be a better place for `nvim` (no need for the `.bin` suffix then)?

Reply:
I just remembered another option entirely.
https://github.com/termux/termux-packages/pull/22343#issuecomment-2489183106